SharedFrameworkValidation.cmd

```diff
@@ -0,0 +1,9 @@
+@if not defined _echo @echo off
```

Comment:
I'm not sure if we will keep this as a script, or turn it into msbuild instead. For now it was easier to just do it like this, but feel free to share opinions if msbuild project + build definition is prefered. Depending on that I will either change it like that, or add a .sh script.